package.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "extension-agregore-renderer",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Mind changing this to 1.2.0?

Reply:
No problem, done. But why not 1.1.0?

Reply:
I think I was still half-asleep and for some reason thought we already had a `1.1` 🤪 Feel free to use that instead of 1.2

Reply:
Okay :)